cookbooks/bcpc-hadoop/templates/default/hb_hbase-site.xml.erb

```diff
@@ -166,4 +156,143 @@
     <value>org.apache.hadoop.hbase.ipc.SecureRpcEngine</value>    
   </property>   
   <% end %>
+  
+  <% if node["bcpc"]["hadoop"]["hbase"]["shortcircuit"]["read"] == true then %>
+  <property>
+    <name>dfs.client.read.shortcircuit</name>
+    <value><%= node["bcpc"]["hadoop"]["hbase"]["shortcircuit"]["read"]%></value>
+  </property>
+
+  <property>
+    <name>dfs.domain.socket.path</name>
+    <value>/var/run/hadoop-hdfs/dn._PORT</value>
+  </property>
+
+  <property>
+    <name>dfs.client.read.shortcircuit.buffer.size</name>
+    <value>131072</value>
+  </property>
+  <% else %>
+  <property>
+    <name>dfs.client.read.shortcircuit</name>
+    <value><%= node["bcpc"]["hadoop"]["hbase"]["shortcircuit"]["read"]%></value>
+  </property>
+  <% end %>
+
+  <property>
+    <name>hbase.regionserver.handler.count</name>
+    <value>128</value>
+  </property>
+
+  <!-- Log responses which take longer than 250ms -->
+  <property>
+    <name>hbase.ipc.warn.response.time</name>
+    <value>250</value>
+  </property>
```

Comment:
Thanks @cbaenziger. Did add a new attribute. Quarter second is a reasonable time to monitor for hbase probably not for a test VM cluster.